docs/install/system_requirements.md

```diff
@@ -18,15 +18,16 @@ SnappyData turns Apache Spark into a mission-critical, elastic scalable in-memor
 
 | Operating System| Version |
 |--------|--------|
-|Red Hat Enterprise Linux|RHEL 6.0 and 7.0|
-|Ubuntu|Ubuntu Server 14.04 and later|
-|CentOS|CentOS 6, 7|
+|Red Hat Enterprise Linux|- RHEL 6.0 </p> - RHEL 7.0 (Recommended Kernel version: 3.10.0-693.2.2.el7.x86_64)|
```

Comment:
change to "Minimum recommended kernet version: ...". Also add JIRA issues numbers below in next line to clarify the reason for the recommendation.